commands/motor/MotorSet.java

```diff
@@ -50,18 +54,22 @@ protected void execute() {
 	}
```

Comment:
the `execute()` function needs to be public (its above this comment in the actual file)

Reply:
out of scope